js/multimodaltextbox/shared/MultimodalTextbox.svelte

```diff
@@ -54,7 +54,10 @@
 	export let file_count: "single" | "multiple" | "directory" = "multiple";
 	export let max_plain_text_length = 1000;
 	export let waveform_settings: Record<string, any>;
-	export let waveform_options: WaveformOptions = {};
+	export let waveform_options: WaveformOptions = {
```

Comment:
`waveform_options` and `waveform_settings` shouldn't even be props here, but when I started refactoring this, the PR got out of hand. Will refactor the audio out of multimodaltextbox here in a separate PR

Reply:
We should add `waveform_options` as a param to `multimodaltextbox`.